### PR TITLE
Add new field to access rpc exec result

### PR DESCRIPTION
**Pull request by koko1123**
cherry picked from this PR: https://github.com/onflow/flow-go/tree/v0.24

### CODE REVIEW

File: integration/go.mod [hunk 1]
```diff
@@ -43,4 +43,5 @@ replace github.com/onflow/flow-go => ../
 
 replace github.com/onflow/flow-go/crypto => ../crypto
 
-replace github.com/onflow/cadence => github.com/dapperlabs/cadence-internal v0.21.1-patch.2
+// v0.21.1-patch.2 seems missing
```
Comment (vishalchangrani) [inline integration/go.mod:46]: so `github.com/dapperlabs/cadence-internal` does exist but is restricted. I can build this branch locally. 
So lets remove this comment.

**Comment by franklywatson** [comment]
Note to merge this when this change is released: https://github.com/onflow/flow/pull/834

File: integration/go.mod (hunk 1 above)
Comment (koko1123) [reply integration/go.mod]: removed!

**Comment by Kay-Zee** [comment]
v0.24 is no longer active, so safe to close

**Pull request closed without merging**